### PR TITLE
Feat add permafk command

### LINKED ISSUE #516: Perm AFK command
If you want to forcefully set yourself as afk regardless if you send a message after the fact for people that maybe want to set afk status and need to message once and don't care to re afk

### PR DESCRIPTION
**Pull request by maukkis**
## Description

This pr contains the feature that allows you to set yourself afk without it going away when you send a message as described in issue #516 the command is permafk and you can disable afk by running the command again

## Guidelines

- My code follows the style guidelines of this project (formatted with Ruff)
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation if needed
- My changes generate no new warnings
- I have tested this change
- Any dependent changes have been merged and published in downstream modules
- I have added all appropriate labels to this PR

- [x ] I have followed all of these guidelines.

## How Has This Been Tested? (if applicable)

tested in my own test server both normal afk command and the permafk command both seem to work fine

## Summary by Sourcery

Add a new 'permafk' command to allow users to set themselves as permanently AFK, preventing the AFK status from being removed when they send a message. Update the database controller to support this feature and enhance the AFK functionality.

New Features:
- Introduce a new 'permafk' command that allows users to set themselves as permanently AFK, which does not reset when they send a message.

Enhancements:
- Add a new method 'is_perma_afk' to check if a user is permanently AFK in the database.

Documentation:
- Update documentation to include information about the new 'permafk' command.

### CODE REVIEW

File: tux/cogs/utility/afk.py [hunk 1]
```diff
@@ -90,7 +90,8 @@ async def remove_afk(self, message: discord.Message) -> None:
 
         if entry.since + timedelta(seconds=10) > datetime.now(ZoneInfo("UTC")):
             return
-
+        if await self.db.is_perma_afk(message.author.id, guild_id=message.guild.id):
```
Comment (kzndotsh) [inline tux/cogs/utility/afk.py]: Change this to perm_afk not perma

File: prisma/schema.prisma [hunk 2]
```diff
@@ -125,6 +125,7 @@ model AFKModel {
   reason    String
   since     DateTime @default(now())
   guild_id  BigInt
+  perma_afk Boolean  @default(false)
```
Comment (kzndotsh) [inline prisma/schema.prisma]: Change this to perm_afk not perma_afk

File: tux/cogs/utility/permafk.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import discord
+from discord.ext import commands
+
+from tux.bot import Tux
+from tux.database.controllers import AfkController
+from tux.utils.constants import CONST
+from tux.utils.flags import generate_usage
+
+
+class PERMAFK(commands.Cog):
```
Comment (kzndotsh) [inline tux/cogs/utility/permafk.py]: Change this to PermAfk

File: tux/database/controllers/afk.py [hunk 4]
```diff
@@ -31,6 +44,7 @@ async def insert_afk(self, member_id: int, nickname: str, reason: str, guild_id:
                 "nickname": nickname,
                 "reason": reason,
                 "guild_id": guild_id,
+                "perma_afk": perma_afk,
```
Comment (kzndotsh) [inline tux/database/controllers/afk.py]: change this to perm_afk

**Review by kzndotsh — CHANGES_REQUESTED** [review]

**Comment by maukkis** [comment]
Fixed those naming issues now.

**Review by kzndotsh — APPROVED** [review]

**Pull request merged**